RULES.md

```diff
@@ -696,6 +696,12 @@ Unimplemented rules from the AWS PCI DSS 3.2.1 Conformance Pack.
 | [ssm-document-not-public](https://docs.aws.amazon.com/config/latest/developerguide/ssm-document-not-public.html)                                                                 | SSM documents are not public.                                                                                                       | 1.2, 1.2.1, 1.3, 1.3.1, 1.3.2, 1.3.4, 1.3.6, 2.2.2                                                                |
 | [vpc-sg-open-only-to-authorized-ports](https://docs.aws.amazon.com/config/latest/developerguide/vpc-sg-open-only-to-authorized-ports.html)                                       | The VPC Security Group restricts IPv4 TCP traffic on unauthorized ports.<sup>[2](#vpc-sg-open-only-to-authorized-ports)</sup>       | 1.2, 1.2.1, 1.3, 1.3.1, 1.3.2, 2.2.2                                                                              |
 
+## Rules not in packs
```

Comment:
I think we should add a reference to the `README` about this. Perhaps under https://github.com/cdklabs/cdk-nag#available-packs section or a related section

Reply:
@dbyod13 this needs to be addressed

Reply:
Apologies, missed that one. Added proposed commit now